Data/SkillSys/SkillInfo.c

```diff
@@ -1838,4 +1838,11 @@ const struct SkillInfo gSkillInfos[0x100] = {
         .icon = GFX_SkillIcon_Corrosion,
     },
 #endif
+
+#if (defined(SID_NoGuard) && (SID_NoGuard < MAX_SKILL_NUM))
+    [SID_NoGuard] = {
+        .desc = MSG_SKILL_NoGuard,
+        .icon = GFX_SkillIcon_WIP,
```

Comment:
I think it is better to add an icon， or find one?

Reply:
Since when was that a barrier to skills being added?